spec/imsc-hrm.html

```diff
@@ -200,7 +200,19 @@
   </section>
 
   <section id='sotd'>
-   
+    <p>For this specification to exit the CR stage, the following independent factors of verifications MUST be demonstrated:</p>
```

Comment:
Suggest only using one of "MUST" and "SHALL" throughout the spec, for consistency. Until now we've preferred "SHALL" but I think "MUST" is a bit easier for most folk, being more commonly used in English, in general.